libs/ui/src/lib/page-banner/page-banner.tsx

```diff
@@ -20,11 +18,7 @@ const PageBanner = ({ title, subtitle, fontColor, backgroundColor, logo, logoDes
         ...(backgroundColor ? { background: backgroundColor } : {}),
       }}
     >
-      {!logo && (
-        <div className={style.svgEllipse}>
-          <EllipseSVG />
-        </div>
-      )}
+      {!logo && <div className={style.svgEllipse}></div>}
```

Comment:
Vil ikke dette gjøre at det blir veldig tomt for de som ikke har lagt inn eget design i admin?